examples/advanced/custom-file-header/README.md

```diff
@@ -0,0 +1,45 @@
+## Custom File Header
+
+Style Dictionary 3.0 added the ability to define custom file headers for files generated from formats. A file header is a comment at the beginning of the generated file that has some information about how it was generated. This is fairly common in build tools that output source code. Before 3.0 you could only show or hide this header in the built-in formats. Now you can write your own custom file header messages that can be used in built-in formats as well as custom formats. 
+
+Common use cases include:
+
+- Using a version number in the file header comment so that you can check in files generated by Style Dictionary to git
+- Using a hash of the source in the file header comment so that it only changes when there is a change in the source
```

Comment:
I think this is what you meant?

> it only changes

it always changes

Reply:
Hopefully this sounds better?